data_steward/cdr_cleaner/manual_cleaning_rules/dc732_suppress.py

```diff
@@ -0,0 +1,445 @@
+# # DC-732 Retract Rows with Suppressed Concepts
+#
+# Identify, log and remove all rows from all tables in specified deid datasets where at least one concept_id field references a concept which must be suppressed.
+#
+# ## Notes
+#  * Prior to running the table identified by `CONCEPT_LOOKUP_TABLE` must be loaded with concepts to suppress. At minimum this table should have the field `concept_id`.
+#  * This can safely be re-run (e.g. if a new concept is added to the lookup).
```

Comment:
nit:  I'm pretty sure these are longer than 80 characters.  :)  Would you mind line wrapping them?